src/ui/features/my-dogs.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { getDogs } from '../../api'
+
+export class MyDogs extends React.PureComponent {
```

Comment:
Creates a react class. I like to reach for "[PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent)" first since it comes with free performance benefits.